graphql/database/missions/utils.js

```diff
@@ -0,0 +1,154 @@
+/* eslint-disable no-nested-ternary */
```

Comment:
Please remove & keep using this ESLint rule 😄

Reply:
this one always gets me, i feel like a simple nested ternary is still human readable and i'm a lazy typer lol